corgi/tasks/managed_services.py

```diff
@@ -0,0 +1,83 @@
+from celery_singleton import Singleton
+from django.db import transaction
+from django.utils import timezone
+
+from config.celery import app
+from corgi.collectors.app_interface import AppInterface
+from corgi.collectors.syft import Syft
+from corgi.core.models import Component, ComponentNode, ProductStream, SoftwareBuild
+from corgi.tasks.common import RETRY_KWARGS, RETRYABLE_ERRORS
+from corgi.tasks.sca import save_component
+
+
+@app.task(base=Singleton, autoretry_for=RETRYABLE_ERRORS, retry_kwargs=RETRY_KWARGS)
+def refresh_service_manifests():
+    services = ProductStream.objects.filter(meta_attr__managed_service_components__isnull=False)
+    service_metadata = AppInterface.fetch_service_metadata(services)
+
+    for service, components in service_metadata.items():
+        manifest_service(service.pk, components)
+        manifest_service.delay(service.pk, components)
+
+
+@app.task(base=Singleton, autoretry_for=RETRYABLE_ERRORS, retry_kwargs=RETRY_KWARGS)
+@transaction.atomic
```

Comment:
I think we don't need to make the task atomic. It will slow down ingestion and I wouldn't expect a partial update to cause much trouble for data readers. We could just pick up the current data on the next run on the task if there was a partial update.